@here/olp-sdk-dataservice-read/lib/client/PartitionsRequest.ts

```diff
@@ -57,4 +58,19 @@ export class PartitionsRequest {
     public getBillingTag(): string | undefined {
         return this.billingTag;
     }
+
+    /**
+     * Setter for the provided partition ids.
```

Comment:
Please add more text. What happens if this is left empty -> we retrieve all partitions.
The user should not provide more than 100 partition Ids -> what happens if he does.

Reply:
Updated